docs/modules/ROOT/pages/using-auth-sdk.adoc

```diff
@@ -77,11 +77,86 @@ AuthServiceConfiguration authServiceConfig = new AuthServiceConfiguration
     .withRedirectUri("org.aerogear.mobile.example:/callback")
     .build();
 
-// You only need to invoke this once every subsequent retrieval of the AuthService 
+// You only need to invoke this once every subsequent retrieval of the AuthService
 // will retrieve the same instance.
 authService.init(context, authServiceConfig);
 ----
 
+==== Authentication Browser
+
+The `AuthService` can also be configured to whitelist or blacklist browsers that should or shouldn't be used during authentication.
```

Comment:
Should we add some context to the difference between whitelist/blacklist or maybe why you would use it?

Reply:
Ya I'll add in some context here